bound/kt/src/test/kotlin/web5/sdk/TestHelper.kt

```diff
@@ -0,0 +1,22 @@
+package web5.sdk
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class UnitTestSuite(name: String) {
+    val tests: MutableList<String>
+
+    init {
+        val path = Paths.get("../../tests/unit_test_cases/$name.json")
```

Comment:
This almost needs to be copied in the binary or at least in the bound directory (kind of like schemas) because if people pull it in maven this would not work..

However, people who pull in maven would not execute theses tests so it could be fine? Just some food for thought

Reply:
fair point, but should be fine for now. as I understand it, our maven artifact does not include tests